docs/src/pages/[platform]/getting-started/troubleshooting/troubleshooting.angular.mdx

```diff
@@ -6,21 +6,12 @@ import { Fragment } from '@/components/Fragment';
   {({ platform }) => import('./shared/angular-polyfill.mdx')}
 </Fragment>
 
+### Could not find a declaration file for module './aws-exports'
 
-### Unhandled Promise rejection console errors
-
-If you see `Unhandled Promise rejection: [message] ; zone: <root> ; ...` errors on console, you can add the following code after you import `zone.js` inside `src/polyfills.ts` file:
-
-```ts{3}
-import 'zone.js/dist/zone'; // Included with Angular CLI.
-
-(window as any).Zone['__zone_symbol__ignoreConsoleErrorUncaughtError'] = true;
-```
-
-### Typescript Strict Mode Errors
-
-Angular 12+ now turns on typescript strict mode by default. This has known type issues when installing Amplify. You can either work around this by disabling strict mode inside your `tsconfig.json`, or by following the TypeScript strict [troubleshooting guide](https://docs.amplify.aws/lib/troubleshooting/strict-mode/q/platform/js/).
```

Comment:
This has been replaced by `Could not find a declaration file for module './aws-exports'` above.